fuzzers/centipede/builder.Dockerfile

```diff
@@ -28,4 +28,4 @@ RUN git clone -n \
     bazel build -c opt :all) && \
   cp "$CENTIPEDE_SRC/bazel-bin/centipede" '/out/centipede'
 
-RUN /clang/bin/clang "$CENTIPEDE_SRC/weak_sancov_stubs.cc" -c -o /lib/weak.o
+RUN cp "$CENTIPEDE_SRC/bazel-bin/libweak_sancov_stubs.so" /out/weak.o
```

Comment:
Here is something I don't fully understand:
In the past, we used to output the `weak.o` generated (by `clang`) to `/lib/` and it was never lost in the container that runs fuzzing.
But here I had to copy `libweak_sancov_stubs.so` to `/out/` instead of `/lib/`, because it will be lost otherwise.
What has changed to cause this difference?

Reply:
Shared object vs static object.
.so is shared object and needs to be used at runtime. Running is done in another image.

Reply:
I see, thanks : )